pkg/plugins/policies/meshhttproute/api/v1alpha1/meshhttproute.go

```diff
@@ -29,6 +29,10 @@ type MeshHTTPRoute struct {
 }
 
 type To struct {
+	// Hostnames limits the effects of the rules to requests to this hostname.
+	// Given hostnames must intersect with the hostname of the listeners the
```

Comment:
I'd expand the comment mentioning that by "listener" we mean both the outbound listener (with implicit "*" hostname) and meshgateway listener

Reply:
clarified about MeshGateway only in the doc string